src/add-to-app/debugging.md

```diff
@@ -30,31 +30,28 @@ remains attached until your `FlutterEngine` is disposed. But you can invoke
 `flutter attach` before starting your engine. `flutter attach` waits for
 the next available Dart VM that is hosted by your engine.
 
-#### Terminal
+## Debug from the Terminal
 
 Run `flutter attach` or `flutter attach -d deviceId` to attach from the terminal.
 
 {% include docs/app-figure.md image="development/add-to-app/debugging/cli-attach.png" caption="flutter attach via terminal" %}
```

Comment:
**chore:** Swap image for text.
````suggestion
```terminal
$ flutter attach
Syncing files to device iPhone 14 Pro...
 7,738ms (!)

:fire: To hot reload changes while running, press "r". To hot restart (and
rebuild state), press "R".

An Observatory debugger and profiler on iPhone 14 Pro is available at:
http://127.0.0.1:65525/EXmCgco5zjo=/
For a more detailed help message, press "h". To detach, press "d"; to quit,
press "q"

```
````